src/backend/distributed/operations/worker_copy_udf.c

```diff
@@ -0,0 +1,70 @@
+/*-------------------------------------------------------------------------
+ *
+ * worker_copy_udf.c
+ *
```

Comment:
missing description

(meta: we should probably do a better job of providing descriptions of what things are for)